### PR TITLE
[ArtemisNG] Use docopt + improvements

### PR DESCRIPTION
**Pull request by mbounabi**
https://jira.kisio.org/browse/NAVP-1383

:warning: You know it but still... This is an other step in a big evolution, comments welcome if you can understand the full scope of the evolution.  Please refrain your urge to put comments everywhere. :warning: 

**Main evolution:** Use docopt to run the script "docker_orchestrator". The options and arguments of the script may/will evolve. TBD in a future refinement maybe, for those interested!

### CODE REVIEW

File: artemis/base_pytest.py [hunk 1]
```diff
@@ -262,6 +262,13 @@ def journey(self, _from, to, datetime,
         for k, v in six.iteritems(kwargs):
             query = "{query}&{k}={v}".format(query=query, k=k, v=v)
 
+        # Override scenario
+        if self.__class__.data_sets[0].scenario in ['distributed', 'experimental', 'asgard']:
+            overridden_scenario = 'experimental'
```
Comment (kinnou02) [inline artemis/base_pytest.py]:
```suggestion
            overridden_scenario = 'distributed'
```
experimental is currently an alias to distributed, but we may want to reuse it later for some experimental feature :)

File: artemis/docker_orchestrator.py [hunk 2]
```diff
@@ -121,12 +151,28 @@ def launch_coverages():
...
+            print("\nTests Done!!!\n")
 
             # Delete instance container
-            kraken_name = "kraken-" + instance_name
-            stopCommand = "TAG=dev docker-compose -f docker-compose.yml -f kirin/docker-compose_kirin.yml -f " + instance_file + " rm -sfv kraken-" + instance_name
-            subprocess.Popen(stopCommand, shell=True)
-            wait_for_docker_stop(kraken_name)
+            # NOTE: the command 'rm -s' doesn't work on version < 1.14.0 (https://github.com/docker/compose/blob/master/CHANGELOG.md)
+
+            # SWITCH: (comment case according to the machine config)
```
Comment (kinnou02) [inline artemis/docker_orchestrator.py]: :nauseated_face: 
could you get the version and choose the appropriate command automatically?
or drop support for docker-compose older than  1.14 it's from 2017 after all

Comment (mbounabi) [reply artemis/docker_orchestrator.py]: If I drop the support on old versions, it won't work on our current Jenkins.
I'll sera what I can do yo cet the running system version

Comment (kinnou02) [reply artemis/docker_orchestrator.py]: Something prevent us to upgrade?

Comment (mbounabi) [reply artemis/docker_orchestrator.py]: The fear of breaking our Jenkins machine...

**Pull request merged**